torch/nn/_intrinsic/qat/modules/conv_bn_relu.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from torch.nn import Conv2d as NNConv2d
+from torch.nn._intrinsic import ConvBn2d as NNConvBn2d
+from torch.nn._intrinsic import ConvBnReLU2d as NNConvBnReLU2d
+from torch.nn.modules.conv import conv2d_forwad
```

Comment:
nit: typo, should be conv2d_forward.  See https://github.com/pytorch/pytorch/pull/22899/files#r305143145